docs/src/pages/_document.page.tsx

```diff
@@ -25,7 +25,7 @@ const getCSPContent = (context: Readonly<HtmlProps>) => {
       style-src 'self' 'unsafe-inline';
       font-src 'self' data:;
       frame-src 'self' *.codesandbox.io;
-      img-src 'self' cm.everesttech.net amazonwebservices.d2.sc.omtrdc.net dpm.demdex.net https://images.unsplash.com;
+      img-src 'self' cm.everesttech.net amazonwebservices.d2.sc.omtrdc.net dpm.demdex.net https://images.unsplash.com data:;
```

Comment:
Not sure if we want to accept all `data:`. @reesscot  and I talked about it and trying to only allow the specific hash, but I didn't figure out how

Reply:
also, the warning is already in `main`, so it might be more clear to resolve it in a separated PR. What do you think?

Reply:
We can remove it, I'll just need to rewrite the inline background image in the CSS :/